jobs/oauth2-proxy/templates/bin/ctl

```diff
@@ -24,6 +24,8 @@ case $1 in
         --client-secret="<%= p('client_secret') %>" \
         --cookie-secret="<%= p('cookie_secret') %>" \
         --email-domain="<%= p('email_domain') %>" \
+        --httpBrowserXssFilter=true \
```

Comment:
WDYT about putting these options in the job spec?

Reply:
Is there any reason why we would want these options ever to be turned off?  Seems like extra yak shaving for something we'll never do.  But if you think this is a good idea, I'm happy to dig in and make that happen.